README.md

```diff
@@ -12,10 +12,23 @@ virtualenv venv
 source venv/bin/activate 
 # Install required python libraries
 pip install -r requirements.txt 
+# Install node modules
+npm install
```

Comment:
@tpllaha 
node edhe gulp duhen vetem per ne(developerat), jo per serverin.
Node duhet per te organizuar kodin ne file te ndryshem ne react. Eshte e bezdishme te punosh pa node ne reactjs. Gulp eshte perdorur per te konvertuar kodin e react-it. `gulp default` e konverton vete sa here ndryshohet nje file nga folderi /jsx/